package.json

```diff
@@ -23,10 +23,10 @@
   ],
   "scripts": {
     "size": "size-limit",
-    "test": "standard && ava",
-    "testci": "standard && ava && npm run size",
+    "testci": "standard &&  nyc ava tap test.js && npm run size",
```

Comment:
Remove `tap` and `test.js`.

Reply:
Ok as i followed codecov i did it based on their reference here what i replied to @stephenmathieson . https://github.com/segmentio/analytics-node/pull/148#discussion_r162044679

So now i am bit confused about what we are implementing here. Codecov OR Coveralls? 

Just let me get back to @f2prateek here about it?

Reply:
My mistake, we're talking about codecov. However, it's still unrelated to this comment.

Reply:
well then i will give it a shot again in same way you suggested. Fact is i tried it for couple of times on my Docker stack but it didn't work so i had to get those back on my local. Once it started to work i replicated same over Cir-CI.

Reply:
It's just `ava tap test.js` is not a valid command, so I'm not sure how it could work for you.